packages/mdc-tabs/tab/_mixins.scss

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@mixin mdc-icon-tab-content {
+  display: block;
+  margin: 0 auto;
+}
+
+@mixin mdc-anchor-reset {
+  color: $mdc-tabs-text-color;
```

Comment:
Since these variables are being referenced without `@import`ing `_variables.scss`, it assumes that the file has already been imported before this is used, which makes the code more brittle. You should `@import './variables'` at the top of this file.